src/test/kotlin/uk/gov/dluhc/notificationsapi/service/TemplateServiceTest.kt

```diff
@@ -454,6 +456,45 @@ class TemplateServiceTest {
         }
     }
 
+    @Nested
+    inner class GenerateBespokeCommTemplatePreview {
+        @ParameterizedTest
+        @EnumSource(value = SourceType::class, names = ["POSTAL", "PROXY", "OVERSEAS", "VOTER_CARD"])
```

Comment:
Could you use your new annotation in src/test/kotlin/uk/gov/dluhc/notificationsapi/testsupport/annotations/SourceTypesApiEnumTest.kt?

Reply:
Oh sorry this is DTO enum not API enum so you can't